test/e2e/send-edit.spec.js

```diff
@@ -1,254 +0,0 @@
-      await driver.delay(regularDelayMs);
-
-      const gasModal = await driver.findElement(By.css('span .modal'));
-
-      const [gasPriceInput, gasLimitInput] = await driver.findElements(
-        By.css('.advanced-gas-inputs__gas-edit-row__input'),
-      );
-
-      await gasPriceInput.clear();
-      await driver.delay(50);
-      await gasPriceInput.sendKeys('10');
-      await driver.delay(50);
-      await driver.delay(tinyDelayMs);
-      await driver.delay(50);
-
-      await gasLimitInput.clear();
-      await driver.delay(50);
-      await gasLimitInput.sendKeys('25000');
-
-      await driver.delay(1000);
-
-      await driver.clickElement(By.xpath(`//button[contains(text(), 'Save')]`));
-      await driver.wait(until.stalenessOf(gasModal));
-      await driver.delay(regularDelayMs);
-
-      // Continue to next screen
-      await driver.clickElement(By.xpath(`//button[contains(text(), 'Next')]`));
-      await driver.delay(regularDelayMs);
-    });
-
-    it('has correct value and fee on the confirm screen the transaction', async function () {
-      const transactionAmounts = await driver.findElements(
-        By.css('.currency-display-component__text'),
-      );
-      const transactionAmount = transactionAmounts[0];
-      assert.equal(await transactionAmount.getText(), '1');
-
-      const transactionFee = transactionAmounts[1];
-      assert.equal(await transactionFee.getText(), '0.00025');
-    });
-
-    it('edits the transaction', async function () {
-      await driver.clickElement(
-        By.css('.confirm-page-container-header__back-button'),
-      );
-
-      await driver.delay(regularDelayMs);
-
-      const inputAmount = await driver.findElement(
-        By.css('.unit-input__input'),
-      );
-
-      await inputAmount.clear();
-      await driver.delay(50);
-      await inputAmount.sendKeys('2.2');
-
-      await driver.clickElement(By.css('.advanced-gas-options-btn'));
-      await driver.delay(regularDelayMs);
-
-      const gasModal = await driver.findElement(By.css('span .modal'));
-
-      const [gasPriceInput, gasLimitInput] = await driver.findElements(
-        By.css('.advanced-gas-inputs__gas-edit-row__input'),
-      );
-
-      await gasPriceInput.clear();
-      await driver.delay(50);
-      await gasPriceInput.sendKeys('8');
-      await driver.delay(50);
-      await driver.delay(tinyDelayMs);
-      await driver.delay(50);
-
-      await gasLimitInput.clear();
-      await driver.delay(50);
-      await gasLimitInput.sendKeys('100000');
-
-      await driver.delay(1000);
-
-      await driver.clickElement(By.xpath(`//button[contains(text(), 'Save')]`));
-      await driver.wait(until.stalenessOf(gasModal));
-      await driver.delay(regularDelayMs);
-
-      await driver.clickElement(By.xpath(`//button[contains(text(), 'Next')]`));
-      await driver.delay(regularDelayMs);
-    });
-
-    it('has correct updated value on the confirm screen the transaction', async function () {
```

Comment:
Nit: It would be nice to at least preserve these descriptions as comments, to make these tests easier to understand.